pages/common/gem.md

```diff
@@ -3,22 +3,34 @@
 > Interact with the package manager for the Ruby programming language.
 > More information: <https://rubygems.org>.
 
+- Display remote gems:
```

Comment:
How about "Search for remote gems" ? And shouldn't it be "gem(s)" here too ?

Reply:
Good suggestion, thanks. [Fixed](https://github.com/tldr-pages/tldr/pull/3172/commits/9c46d8aeedc702cacb62b158a6f900fbca0b544e).